routes/index.js

```diff
@@ -140,7 +140,7 @@ router.post(
  *   }
  */
 router.post(
-  '/:address/disputes/:disputeHash',
+  '/:address/disputes/:arbitratorAddress/:disputeId',
```

Comment:
it's maybe more REST : `/:address/arbitrator/:arbitratorAddress/disputes/:disputeId`

Reply:
good idea